addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -4788,9 +4788,9 @@ registry.layout_column = SnippetOptionWidget.extend({
         var colOffset = Math.floor((12 - colSize * colsLength) / 2);
         var colClass = 'col-lg-' + colSize;
         _.each($columns, function (column) {
-            var $column = $(column);
-            $column.attr('class', $column.attr('class').replace(/\b(col|offset)-lg(-\d+)?\b/g, ''));
-            $column.addClass(colClass);
+            let className = column.className.replace(/\b(col|offset)-lg(-\d+)?\b/g, "");
+            className += (className ? " " : "") + colClass;
+            column.className = className;
```

Comment:
Same remark: we should not have to care about this (well, we can to make more pretty code, but the current implementation should work). I would be very surprised if there were actually only two occurences of some code doing that... and if you consider community apps & custos... :grimacing: